### DISCUSSION TITLE
Welcome to Registry Discussions!

### DISCUSSION THREAD

**Discussion by EricZimmerman** [EricZimmerman/Registry#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JohnnyCena123** [comment]
you saved me today. I was reinstalling windows, I backed up my NTUSER.DAT, and when I went to load the hive in regedit, basically everything was gone. when I opened it in this registry explorer, I found out why - it was all marked as deleted. thanks to this amazing utility I managed to restore all of my registry. thanks!

**Comment by EricZimmerman** [comment]
https://www.youtube.com/watch?v=XgUB3lF9IQA

Glad it was helpful! I've never heard of backing up a hive and reusing it on a clean install.
